edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/WorkloadClient.cs

```diff
@@ -37,7 +37,7 @@ internal WorkloadClientVersioned GetVersionedWorkloadClient(Uri workloadUri, str
                 return new Version_2018_06_28.WorkloadClient(workloadUri, supportedVersion, moduleId, moduleGenerationId);
             }
 
-            if (supportedVersion == ApiVersion.Version20190130)
+            if (supportedVersion.CompareTo(ApiVersion.Version20190130) >= 0)

```

Comment:
also check not to be unknown because unknown should still failover to ersion_2018_06_28